src/mmw/js/src/analyze/controllers.js

```diff
@@ -49,12 +49,15 @@ var AnalyzeController = {
 
     analyze: function() {
         var aoi = JSON.stringify(App.map.get('areaOfInterest')),
-            analyzeModel = createTaskModel(aoi),
+            analyzeModel = App.analyzeModel || createTaskModel(aoi),
             analyzeResults = new views.ResultsView({
                 model: analyzeModel
             });
 
-        App.analyzeModel = analyzeModel;
+        if (!App.analyzeModel) {
+            App.analyzeModel = analyzeModel;
+        }
+
```

Comment:
Consider encapsulating this logic in a utility function associated with the App.